docs/_user_guide/best_practices_migrating_from_jupyter.md

```diff
@@ -50,26 +55,27 @@ The biggest hurdle in migrating your process from a JupyterHub notebook to `yen-
 * Write a submit script to run your process on `yen-slurm` using your working environment
 
 
-## Python Virtual Environments
+### Python Virtual Environments
 
-For Python, you can use  `venv` to create an environment that can be shared across Yen1-5, `yen-slurm`, and JupyterHub. See [this page](/topicGuides/pythonEnv.html) for information on setting up a Python virtual environment using `venv`.
+For Python, you can use  `venv` to create an environment that can be shared across Yen1-5, `yen-slurm`, and JupyterHub. See [this page](/_user_guide/best_practices_python_env/){target="_blank"} for information on setting up a Python virtual environment using `venv`.
 
 #### Activate your environment and install ipykernel
 
-```bash
-$ source <path/to/your/venv>/bin/activate
-$ pip install ipykernel
+```title="Terminal Command"
+source <path/to/your/venv>/bin/activate
+pip install ipykernel
 ```
 The `ipykernel` package is required to make a virtual environment into a JupyterHub kernel.
 
 #### Setup that environment in JupyterHub
 
-{% include important.html content="Make sure your environment is **active** (`source <path/to/your/venv>/bin/activate`) before installing it on JupyterHub!" %}
+!!! important
+    Make sure your environment is **active** (`source <path/to/your/venv>/bin/activate`) before installing it on JupyterHub!
```

Comment:
can probably delete the command in parentheses. I think it's clear what active env means

Reply:
done